lib/index.js

```diff
@@ -1,4 +1,70 @@
 // This is the base url
 // make requests with `herokuUrl + "api/v1/foods"` etc
 const herokuUrl = require("./heroku-url")
-const food = require('./foods')
+const $ = require("jQuery")
+const food = require('./foods');
+
+//$('.newMealFoodForm').hide()
+
+function getMeals() {
+  $.get(`${herokuUrl()}/api/v1/meals`)
+    .then(function(meals) {
+      let totals = {}
+      let remainingCalories= {
+        Breakfast: 400,
+        Lunch: 600,
+        Dinner: 800,
+        Snack: 200
+      }
```

Comment:
@bretfunk Are these default amounts or do we want to let the user define their own meal goals for calories?

Reply:
@bretfunk Summary of our conversation just now: 
- We may look into making the `goalCalories` of a meal to have default values that can be over written.
- Or, we auto fill with these values and but make the `goal` box be editable so a user can customize their meals!